migrate.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Creating aliases for indizes based on a name pattern,
+# like it's common in ELK/EFK setups
+
+# Fail when using unset variables
+set -u
+
+# Exit in case of error
+set -e
+
+# Number of days to expect indices for
+num_days=16
+
+# Date format used in our index pattern
+format="+%Y.%m.%d"
+
+oldprefix="fluentd"
```

Comment:
could we extract those value in env too to make them param?

Reply:
i mean old and new prefix

Reply:
Yeah, will do that.